Cargo.toml

```diff
@@ -33,7 +33,7 @@ waker-fn = "1.1.0"
 autocfg = "1"
 
 [target."cfg(unix)".dependencies]
-libc = "0.2.77"
+rustix = { version = "0.36.0", features = ["fs"] }
```

Comment:
Is it possible to disable the `use-libc-auxv` feature?

Reply:
Yes; I've now added a patch that does that.